test/05-tag-locking.c

```diff
@@ -41,7 +39,7 @@ main(int argc, char** argv)
     //plc_tag_set_debug_level(PLCTAG_DEBUG_SPEW);
 
     for (i = 0; i < sizeof(threads) / sizeof(threads[0]); i++) {
-        if (pthread_create(&threads[i], NULL, thread_entry, NULL)) {
+        if (pthread_create(&threads[i], NULL, thread_entry, (uintptr_t)i)) {
```

Comment:
This is warning on clang:

```
05-tag-locking.c:42:61: warning: incompatible integer to pointer conversion passing 'uintptr_t' (aka 'unsigned long') to
      parameter of type 'void *' [-Wint-conversion]
        if (pthread_create(&threads[i], NULL, thread_entry, (uintptr_t)i)) {
```

Reply:
Fixed, thanks.

Reply:
Also removed the Makefiles - typed too fast and didn't realize `git --rm` had failed. Apparently I can't see stderr today...